app/javascript/entrypoints/photo_gallery.js

```diff
@@ -1,45 +1,77 @@
-// var photoArray = [];
-//
-// document.addEventListener("DOMContentLoaded", function () {
-//   photoArray = [];
-//   let gallery = photoSwipe();
-//   let photoSlide = document.getElementById("photo-slide");
-//   if (photoSlide) {
-//     photoSlide.childNodes.forEach(function (node) {
-//       let img = node;
-//       if (img.tagName === "IMG") {
-//         photoArray.push({
-//           src: img.getAttribute("src"),
-//           w: img.getAttribute("data-width"),
-//           h: img.getAttribute("data-height"),
-//         });
-//         img.addEventListener("click", function () {
-//           gallery.init();
-//         });
-//       }
-//     });
-//   }
-//   let showPhoto = document.getElementById("show-photo");
-//   if (showPhoto) {
-//     showPhoto.addEventListener("click", function () {
-//       gallery.init();
-//     });
-//   }
-// });
-//
-// function photoSwipe() {
-//   var pswpElement = document.querySelectorAll(".pswp")[0];
-//
-//   var items = photoArray;
-//   var options = {
-//     index: 0,
-//   };
-//
-//   var gallery = new PhotoSwipe(
-//     pswpElement,
-//     PhotoSwipeUI_Default,
-//     items,
-//     options
-//   );
-//   return gallery;
-// }
+import PhotoSwipe from "photoswipe";
+import PhotoSwipeUI_Default from "photoswipe/dist/photoswipe-ui-default";
+
+document.addEventListener("DOMContentLoaded", function () {
+  var images = [];
```

Comment:
Please use let or const depending on use cases, do not use var anymore. Fix in the whole file please